symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -386,4 +399,25 @@ private static Long getEpochMillis(Instant instant) {
   private <T> T executeAndRetry(String name, SupplierWithApiException<T> supplier) {
     return RetryWithRecovery.executeAndRetry(retryBuilder, name, supplier);
   }
+
+
+  private File createTemporaryAttachmentFile(Attachment attachment) {
+    File tempFile;
+    if (attachment != null) {
```

Comment:
Having an early return will slightly increase readability in my opinion, like:
`if (attachment == null) { return null; }` and then the regular case.